scripts/nesi_scripts/adv_im/run_adv_im.sh

```diff
@@ -224,24 +239,27 @@ else
     sed -i "/threshold_time = /c\    threshold_time = $WCT_BENCHMARK_MAXHOUR" $gmsim/workflow/estimation/estimate_wct.py
 fi
 
-
+#################################################################
+# create automated submission socket
+#################################################################
 create_runsocket
 while [[ $? != 0 ]]; 
 do
     echo "failed to start up screen socket for automated submission"
-    ask_yn "re-attempt to create screen in 5 seconds?" 'sleep 5;eval create_runsocketi;' 'exit 5;'
+    ask_yn "re-attempt to create screen in 5 seconds?" 'sleep 5;create_runsocket;' 'exit 5;'
 done
 
-###### Observed related steps ######
-
+#################################################################
```

Comment:
I think it's probably worth having a separate workflow for observed rather than including them in simulation. Because we should be only running observed once rather than for multiple simulations.

Reply:
Thoughts @jpa198